tests/CI/docker-compose.yml

```diff
@@ -101,6 +101,8 @@ services:
       - DIRACX_CONFIG_BACKEND_URL=git+file:///cs_store/initialRepo
       - DIRACX_SERVICE_AUTH_TOKEN_KEY=file:///signing-key/rs256.key
     pull_policy: always
+    command: ["sleep", "infinity"]
```

Comment:
Reference about this: issue is described in https://github.com/moby/moby/issues/42275 and seems like https://github.com/moby/moby/issues/42275#issuecomment-1616442602 would contain some good suggestions, but just ran into other issues. The above is a hack/workaround.

Reply:
you may want to add that comment directly into the `docker-compose` file, it's very useful !